sceptre/config/reader.py

```diff
@@ -401,7 +401,7 @@ def _collect_s3_details(stack_name, config):
         s3_details = None
         if "template_bucket_name" in config:
             template_key = "/".join([
-                stack_name, "{time_stamp}.json".format(
+                stack_name.replace("\\", "/"), "{time_stamp}.json".format(
```

Comment:
One small thing here is you can use `sceptreise_path` for stack_name.

e.g.

```python 
template_key = "/".join([
    sceptreise_path(stack_name), "{time_stamp}.json".format(
        time_stamp=datetime.datetime.utcnow().strftime(
            "%Y-%m-%d-%H-%M-%S-%fZ" 
)
```


This way you get test coverage for free ;)